distributor-api-v1/operations.md

```diff
@@ -151,20 +151,107 @@ Preferred initial call used to obtain all static data about distributor configur
+
+#### PaymentCardInput
+
+* `NotRequested` - enterprise does not want to request payment card info.
+* `Requested` - enterprise wants a field for customers to enter their card details into, but does not want the card details to be validated.
+* `Required` - enterprise wants a field for customers to enter their card details into but the card must be validated before the reservation can be created. 
+
+#### RequiredField
+
+* `Telephone` - telephone field is required for both [Customer](operations.md#customer) and [Booker](operations.md#booker)
+
+#### Enterprise 
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `Id` | string | required | Unique identifier of the an enterprise. |
+| `Name` | [LocalizedText](operations.md#localizedtext) | required | Enterprise name. |
+| `Description` | [LocalizedText](operations.md#localizedtext) | required | Enterprise Description. |
+| `Address` | [Address](operation.md#address) | required | Address of the enterprise. |
+| `CityId` | string | required | Unique identifier of the city enterprise is in. |
+| `ImageId` | string | optional | Unique identifier of logo of the enterprise. |
+| `IntroImageId` | string | optional | Unique identifier of an image that enterprise set up as login screen image. |
+| `Email` | string | required | Email of the enterprise. |
+| `Telephone` | string | required | Telephone of the enterprise. |
+| `TermsAndConditionsUrl` | [LocalizedText](operations.md#localizedtext) | required | Url with Terms and Conditions of the enterprise. |
+| `PrivacyPolicyUrl` | [LocalizedText](operations.md#localizedtext) | required | Url with Privacy policy of the enterprise. |
+| `DefaultLanguageCode` | string | required | Language code of default language for the Enterprise. |
+| `DefaultCurrencyCode` | string | required | Currency code of default currency for the Enterprise. |
+| `DefaultRateCurrencyCode` | string | required | Currency code of default rate for the Enterprise. |
+| `IanaTimeZoneIdentifier` | string | required | Iana time zone identifer. |
+| `AcceptedCurrencyCodes` | array of string | required | List of currency codes accepted by Enterprise. |
+| `TaxEnvironmentCode` | string | required | Tax environment code. |
+| `Pricing` | string [Pricing](operations.md#pricing) | required | Pricing method that enterprise uses. |
+| `Categories` | array of [RoomCategory](operations.md#roomcategory) | required | List of active room categories of the Enterprise. |
+| `Products` | array of [Product](operations.md#product) | required | List of active products that are available to be offered the customer. |
+| `AdditionalLegalStatements` | array of [LocalizedText](operations.md#localizedtext) | required | Additional legal statements. |
+
+#### Address
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Line1` | string | optional | First line of the address. |
```

Comment:
Line1 is really optional?

Reply:
Based on this:
`Line1 = address.AddressLine1.GetOrNull()`
`src/Mews.Server.Web/Api/Distributor/V1/DistributorApiConfiguration.cs` it seems so.

Or is that not correct?

Reply:
tried it in commander and it can be